chapters/infrastructure/processing/reading/README.md

```diff
@@ -1,12 +1,163 @@
-### Content Processing
+# Content Processing
 
-Owner: Vlad Năstase
-Reviewer: Teo Duțu
+Educational content organized in a Version Control System like `git`, and published to a `GitHub` repository is suitable for collaborative development.
+However, it might be a challenging medium for learners to interact with directly.
+As a consequence, the content should be presented in a much more friendly and interactive manner.
+To that end, the content is processed so that it is deliverable to a publisher (website) in a more approachable way.
 
-process content to render it to the end user (learners)
+The types of content that need to be processed vary with course requirements and require different actions in order to be rendered.
+The various types of content are defined and explained in the [Developing and Organizing Content](../../../develop-organize/overview/reading/README.md) chapter.
 
-different types of content to be homogenized
+The rendered output that we recommend is a website generated using a static website generator.
+The usage of [Docusaurus](https://docusaurus.io/) is also recommended, but other static website generators ([Hugo](https://gohugo.io/), [Docsy](https://www.docsy.dev/), [Sphinx](https://www.sphinx-doc.org/en/master/)) can be used.
+The rendered website can embed materials such as slides and quizzes, and can also refer the end user/learner to the `GitHub` repository from which the content is generated (or to an auxiliary repository) for practical content, such as drills.
 
-site + file archives / repositories to be present to learners
+Most static website generators have a documented structure that needs to be followed for the website to render.
+This can be incompatible with the way content is organized for easier collaboration.
 
-OpenEdu builder
+To facilitate content processing with the scope of being published, we have developed the [`Open Education Builder`](https://open-education-hub.github.io/openedu-builder/).
+
+## Open Education Builder
+
+The Open Education Builder is an application that helps organize, transform, and present content.
+It is built with a modular architecture in mind, the smallest building block being the `plugin`.
+A `plugin` represents a simple script that transforms the original content in some way.
+A few examples of such plugins are the ones developed for the technologies recommended in our methodology:
+
+- [`RevealMD`](https://open-education-hub.github.io/openedu-builder/plugins/revealmd) - builds multiple slides that are based on `reveal-md`;
+- [`Docusaurus`](https://open-education-hub.github.io/openedu-builder/plugins/docusaurus) - translates the content so that the structures matches the input expectations of Docusaurus.
```

Comment:
```suggestion
- [`Docusaurus`](https://open-education-hub.github.io/openedu-builder/plugins/docusaurus) - translates the content so that the structures match the input expectations of `Docusaurus`.
```